deploy/crds/pulumi.com_stacks.yaml

```diff
@@ -80,6 +80,105 @@ spec:
                   which can be optionally specified inline. If this is omitted, configuration
                   is assumed to be checked in and taken from the source repository.
                 type: object
+              configRefs:
+                additionalProperties:
+                  description: ConfigRef identifies a resource from which config information
+                    can be loaded. Environment variables, files on the filesystem,
+                    Kubernetes Secrets, ConfigMaps, structured and config literal
+                    values strings are currently supported.
+                  properties:
+                    configmap:
```

Comment:
Interesting. It could be an array, but do we really need this? 🤔 `ConfigRefs` would have the same semantics as `EnvRefs` field.

In EnvRefs, one could specify more than one secret, or more than one env var, with the same key name and the order will not be guaranteed; but this would look like a misconfiguration to me (using the same secret key twice), the behavior seems reasonable.

also, the `ConfigRefs` spec above supports to specify the same ref type more than once, like:

```
configRefs:
   first-secret:
      type: Secret
      secret:
        name: my-secret
        key: key-name
   second-secret:
      type: Secret
      secret:
        name: my-another-secret
        key: key-name
```

Reply:
Ok, it makes sense. I will change it 👍

Reply:
Thinking about the best way to do it 🤔 ...maybe an "array of maps"?

```yaml
configRefs:
   - base:
        type: Structured 
        value:
           my-config: base-value
   - override:
        type: Structured
        value:
           my-config: override-value
```

Or an object with a "name" field (or "key" or something)?

```yaml
configRefs:
   - name: base
      type: Structured 
      value:
         my-config: base-value
   - name: override
      type: Structured
      value:
         my-config: override-value
```

I prefer the first one but any thoughts?

Reply:
@blampe just added a new test case for this scenario